cmd/enroll.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+type enrollTokenData struct {
+	Token string `json:"token"`
+}
+
+type enrollResponse struct {
+	Success bool            `json:"success"`
+	Message string          `json:"message"`
+	Data    enrollTokenData `json:"data"`
+}
+
+func getApiUrl(firstLetter string) (*string, error) {
```

Comment:
Go convention is to use [all caps for acronyms](https://google.github.io/styleguide/go/decisions#initialisms)
```suggestion
func getAPIURL(firstLetter string) (*string, error) {
```